src/components/3_Quadrant/RestaurantList.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const RestaurantList = ({ restaurants }) => {
+
+  return (
+    <>
+      {
+        restaurants
```

Comment:
make it so your restaurants selector (or reducer) has restaurants as an empty array instead of undefined or null. That will make it so you don't need this ternary.

Reply:
THANK YOU! This suggestion has helped us with a big issue throughout the frontend =D